### DISCUSSION TITLE
Series markers

### DISCUSSION THREAD

**Discussion by OccVeneficus** [Live-Charts/LiveCharts2#1060] [Ideas]
Hi, first of all, I would like to thank all the contributors for the work you do. It's very impressive, and I appreciate your efforts.

My suggestion is to add built-in markers for graph lines, as shown in the example. 

**Sample of markers that I'm talking about:**
![marker sample](https://github.com/beto-rodriguez/LiveCharts2/assets/59768959/e4ed9661-8897-4823-b703-ae1329b6296d)

Alternatively, tools for easily creating similar markers could be considered. Is there something similar already available or planned?
What existing tools could be used to implement such functionality?

Currently we considering using this library, and I want to know, if we need to create our own markers based on existing instruments (annotatons, lables, mouse events etc.) or similar feature will be added in upcoming releases and we just need to wait.

**Comment by beto-rodriguez** [comment]
> Is there something similar already available or planned?

This is an interesting feature; this will be surely added in a future.

> What existing tools could be used to implement such functionality?

Visual elements, you can create complex visual elements, but there are no good docs about it yet, but the repo contains many interesting samples that maybe you can find a way to figure out.

https://github.com/beto-rodriguez/LiveCharts2/tree/master/samples/ViewModelsSamples/General/VisualElements

**Reply by OccVeneficus** [reply]
> this will be surely added in a future.

Good to hear that, but i think this isn't something we should expect in nearest future, right?
> Visual elements, you can create complex visual elements

Thank you, i will look into that more.

**Reply by OccVeneficus** [reply]
As i understand there is no movable by mouse elements right now? We did some markers in one of our projects were customer had scichart license - we used groups of three annotaions - polygone triangle, line and text annotaion, but in scichart annotations can be dragged by mouse, wich made moving marker based on wich part of it were dragged relativly simple, using the hit test events and mouse cursor coordinate  (similar logic to livechart AddPointOnClick sample). If visual elements in livecharts had drag behaviour it would really help to create markers.
